java/src/main/java/org/eclipse/ditto/client/management/CommonManagement.java

```diff
@@ -165,6 +164,35 @@
      */
     CompletableFuture<Thing> create(Thing thing, Option<?>... options);
 
+    /**
+     * Creates a {@link Thing} based on the given {@link JsonObject}.
+     *
+     * @param thing a JSON object representation of the Thing to be created. The provided JSON object is required to
+     * contain a field named {@code "thingId"} of the basic JSON type String which contains the identifier of the Thing
+     * to be created. It must conform to the namespaced entity ID notation (see Ditto documentation).
+     * @param options options to be applied configuring behaviour of this method, see {@link
+     * org.eclipse.ditto.client.options.Options}.
+     * @return completable future providing the created Thing object or a specific {@link
+     * org.eclipse.ditto.model.base.exceptions.DittoRuntimeException} if the operation failed
+     * @throws IllegalArgumentException if {@code thing} is {@code null} or if it does not contain the field named
+     * {@code "thingId"}.
+     * @throws org.eclipse.ditto.model.base.exceptions.DittoJsonException if {@code thing} cannot be parsed to a {@link
+     * Thing}.
+     * @throws org.eclipse.ditto.model.things.ThingIdInvalidException if the {@code thingId} was invalid.
+     */
+    CompletableFuture<Thing> create(JsonObject thing, Option<?>... options);
+
+    /**
+     * Creates an empty {@link Thing} with an auto-generated identifier.
+     *
+     * @param options options to be applied configuring behaviour of this method, see {@link
+     * org.eclipse.ditto.client.options.Options}.
+     * @param initialPolicy a custom policy to use for the Thing instead of the default Policy.
```

Comment:
the `intialPolicy` param should be documented before the `options` param - in order of their appearance in the signature